integration/basic_test.go

```diff
@@ -128,22 +128,74 @@ func TestFindCommentQuery(t *testing.T) {
 	assert.Contains(t, databaseNames, name)
 }
 
-func TestLongCollectionName(t *testing.T) {
+func TestCollectionName(t *testing.T) {
 	t.Parallel()
 
-	ctx, collection := Setup(t)
-
-	longCollectionName := "very_long_collection_name_that_is_more_than_64_characters_long_but_still_valid"
-	err := collection.Database().CreateCollection(ctx, longCollectionName)
-	require.NoError(t, err)
-
-	sixtyThreeCharsCollectionName := "this_is_a_collection_name_that_is_63_characters_long_abcdefghij"
-	err = collection.Database().CreateCollection(ctx, sixtyThreeCharsCollectionName)
-	require.NoError(t, err)
-
-	names, err := collection.Database().ListCollectionNames(ctx, bson.D{})
-	require.NoError(t, err)
-
-	assert.Contains(t, names, longCollectionName)
-	assert.Contains(t, names, sixtyThreeCharsCollectionName)
+	t.Run("Err", func(t *testing.T) {
+		ctx, collection := Setup(t)
+
+		cases := map[string]struct {
+			collection string
+			err        *mongo.CommandError
+			alt        string
+		}{
+			"TooLongForBoth": {
+				collection: "very_long_collection_name_that_fails_both_in_mongo_and_in_ferretdb_databases" +
+					"_for_ferretdb_it_fails_because_it_is_more_than_119_characters__for_mongo_it_fails_because_it_is_more_than_255_charachters_" +
+					"long_that_excludes_non_latin_letters_spaces_dots_dollars_dashes",
```

Comment:
Unless we really want people to actually read it, it could be replaced with `const collectionName300 = strings.Repeat("a", 300)` and referring to it both there and below, in the error message

The same in other places

Reply:
> Unless we really want people to actually read it, it could be replaced with `const collectionName300 = strings.Repeat("a", 300)` and referring to it both there and below, in the error message
> 
> The same in other places

Changed here and in other places for collection name tests.